examples/projections/nongeo/polar-misc.py

```diff
@@ -0,0 +1,155 @@
+
+fig.shift_origin(xshift="8c")
+
+# ============
+fig.basemap(
+    # set map limits to theta_min = 0, theta_max = 360, radius_min = 0, radius_max = 1
+    region=[0, 360, 0, 1],
+    # set map width to 5 cm and rotate input data clockwise relative to north direction
+    projection="P5c+a",
+    # set the figures frame and color
+    frame=["xa45f", "+gbisque"],
+)
+
+fig.text(position="TC", text="projection='P5c+a'", offset="0/2.0c", no_clip=True)
+
+fig.text(position="TC", text="region=[0, 360, 0, 1]", offset="0/1.5c", no_clip=True)
+
+fig.shift_origin(xshift="8c")
+
+# ============
+fig.basemap(
+    # set map limits to theta_min = 0, theta_max = 90, radius_min = 0, radius_max = 1
+    region=[0, 90, 0, 1],
+    # set map width to 5 cm and rotate input data clockwise relative to north direction
+    projection="P5c+a",
+    # set the figures frame and color
+    frame=["xa45f", "ya0.2", "WNe+gbisque"],
+)
+
+fig.text(position="TC", text="projection='P5c+a'", offset="0/2.0c", no_clip=True)
+
+fig.text(position="TC", text="region=[0, 90, 0, 1]", offset="0/1.5c", no_clip=True)
+
+fig.shift_origin(xshift="-16c", yshift="-7c")
+
+# ============
+fig.basemap(
+    # set map limits to theta_min = 0, theta_max = 90, radius_min = 0, radius_max = 1
+    region=[0, 90, 0, 1],
+    # set map width to 5 cm and rotate input data clockwise relative to north direction,
+    # rotate coordinate system counterclockwise by 45 degrees
+    projection="P5c+a+t45",
+    # set the figures frame and color
+    frame=["xa45f", "ya0.2", "WNe+gbisque"],
+)
+
+fig.text(position="TC", text="projection='P5c+a\+t45'", offset="0/2.0c", no_clip=True)
+
+fig.text(position="TC", text="region=[0, 90, 0, 1]", offset="0/1.5c", no_clip=True)
+
+fig.shift_origin(xshift="8c", yshift="1.3c")
+
+# ============
+fig.basemap(
+    # set map limits to theta_min = 0, theta_max = 90, radius_min = 3480,
+    # radius_max = 6371 (Earth's radius)
+    region=[0, 90, 3480, 6371],
+    # set map width to 5 cm and rotate input data clockwise relative to north direction,
+    # rotate coordinate system counterclockwise by 45 degrees
+    projection="P5c+a+t45",
+    # set the figures frame and color
+    frame=["xa45f", "ya", "WNse+gbisque"],
+)
+
+fig.text(position="TC", text="projection='P5c+a\+t45'", offset="0/2.0c", no_clip=True)
+
+fig.text(
+    position="TC", text="region=[0, 90, 3480, 6371]", offset="0/1.5c", no_clip=True
+)
+
+fig.shift_origin(xshift="8c")
+
+# ============
+fig.basemap(
+    # set map limits to theta_min = 0, theta_max = 90, radius_min = 3480,
+    # radius_max = 6371 (Earth's radius)
+    region=[0, 90, 3480, 6371],
+    # set map width to 5 cm and rotate input data clockwise relative to north direction,
+    # rotate coordinate system counterclockwise by 45 degrees,
+    # r axis is marked as depth
+    projection="P5c+a+t45+z",
+    # set the figures frame and color
+    frame=["xa45f", "ya", "WNse+gbisque"],
```

Comment:
`a30` may look better
```suggestion
    frame=["xa30f", "ya", "WNse+gbisque"],
```